### PR TITLE
its --> it's in worker.py test

### PR DESCRIPTION
**Pull request by akkatracker**

### CODE REVIEW

File: test/worker.py [hunk 1]
```diff
@@ -480,8 +480,8 @@ def verify_successful_worker_copy_with_reparent(self, mysql_down=False):
           condition_fn=lambda v: v.get('WorkerRetryCount') >= 2)
       logging.debug('Worker has retried at least twice, starting reparent now')
 
-      # vtworker is blocked at this point. This is a good time to test that its
-      # throttler server is reacting to RPCs.
+      # vtworker is blocked at this point. This is a good time to test that 
+      # it's throttler server is reacting to RPCs.
```
Comment (alainjobart) [inline test/worker.py]: this one was actually correct. :)

Comment (akkatracker) [inline test/worker.py]: Oops...will fix :)

Comment (akkatracker) [inline test/worker.py]: @alainjobart all fixed

**Comment by michael-berlin** [comment]
Thanks! :) I just merged the change.

**Pull request merged**